serverless/src/shared/fetchEdlClientToken.js

```diff
@@ -1,30 +1,63 @@
 import { getEdlConfig } from '@/shared/getConfig'
 
+import { logger } from './logger'
+
 /**
  * The EDL client token is used for retrieving/modifying user/groups in URS.
  * @returns the EDL client token
  */
 const fetchEdlClientToken = async () => {
+  logger.debug('Starting to fetch EDL client token')
+
   const { host, uid } = getEdlConfig()
+  logger.debug('EDL host:', host)
+  logger.debug('EDL UID:', uid)
 
   const { EDL_PASSWORD: password } = process.env
+  logger.debug('EDL password:', password ? 'Present' : 'Not present')
 
   const url = `${host}/oauth/token`
+  logger.debug('Token URL:', url)
+
   const authorizationHeader = `Basic ${Buffer.from(`${uid}:${password}`).toString('base64')}`
+  logger.debug('Authorization header:', `${authorizationHeader.substring(0, 20)}...`)
```

Comment:
Is this just going to print out the header key? Because we should not print out the actual header value for these